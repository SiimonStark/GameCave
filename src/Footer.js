import React from 'react';
import GenreBar from './GenreBar';
import PlatformBar from './PlatformBar';
import LuckyBar from './LuckyBar';
import SearchBar from './SearchBar';
import MultiBar from './MultiBar';

export default function Footer(props) {
	return (
		<div className='footer'>
			<SearchBar {...props}/>
			<PlatformBar {...props}/>
			<GenreBar {...props} />
			<MultiBar {...props}/>
			<LuckyBar {...props}/>
		</div>
	);
}
