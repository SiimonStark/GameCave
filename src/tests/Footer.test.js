import React from 'react';
import ReactDOM from 'react-dom';
import Footer from '../Footer';
import { shallow } from 'enzyme';

describe('Footer', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Footer />
    );
  });
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
})