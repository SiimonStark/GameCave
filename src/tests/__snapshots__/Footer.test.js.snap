// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<div
  className="footer"
>
  <SearchBar />
  <PlatformBar />
  <GenreBar />
  <MultiBar />
  <LuckyBar />
</div>
`;
